src/test/kotlin/CalculateTest.kt

```diff
@@ -0,0 +1,14 @@
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class SumTest {
+
+  @Test
+  fun testCalculate() {
+    val sum = Sum()
+    val result = sum.calculate()
+    assertEquals(23, result, "The calculate function should add multiples of 3 or 5 for anything under the number 1000")
```

Comment:
Does the test pass? A sum of number multiple by 3 or 5 under the number 1000 is way more than 23